docs: Fixes to GPU/NVIDIA guide (#1318)
canonical/ubuntu-pro-for-wsl: docs/howto/gpu-cuda.md

## What you will need

The following steps assume a specific hardware configuration. Although the concepts are essentially the same for other architectures, different hardware configurations will require the appropriate graphics drivers and CUDA toolkit.

Make sure the following prerequisites are met before moving forward:

* A Windows 10 version 21H2 or newer physical machine equipped with an NVIDIA graphics card and administrative permission to be able to install device drivers
* Ubuntu 20.04 or higher installed on WSL 2

> * Familiarity with Linux command line utilities and interacting with Ubuntu on WSL2

Reviewer: ```suggestion
* Familiarity with Linux command line utilities and interacting with Ubuntu on WSL 2
```

We are currently not consistent about this, but "WSL 2" (with a space) is used elsewhere in this guide.

* Broadband internet connection able to download a few GB of data

```{note}
If you need more introductory topics, such as how to install Ubuntu on WSL, refer to previous tutorials that can be found [here](../howto/install-ubuntu-wsl2.md).

```

## Install the appropriate Windows vGPU driver for WSL

```{note}
 Specific drivers are needed to enable use of a virtual GPU, which is how Ubuntu applications are able to 
 
 access your GPU hardware, so you’ll need to follow this step even if your system drivers are up-to-date.

```

Please refer to the official [WSL documentation](https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps) for up-to-date links matching your specific GPU vendor. You can find these in `Install support for Linux GUI apps > Prerequisites` . For this example, we will download the `NVIDIA GPU Driver for WSL`.

![Install support for Linux GUI apps page on Microsoft WSL documentation.](assets/gpu-cuda/install-drivers.png)

```{note}

> 

Reviewer: ```suggestion
```

Space not needed here, or in other blocks like this.

This is the only device driver you’ll need to install. Do not install any display driver on Ubuntu.

> 

Reviewer: ```suggestion
```

Space not needed here either, for example.

## Install NVIDIA CUDA on Ubuntu

```{note}

> 
> Normally, CUDA toolkit for Linux will have the device driver for the GPU packaged with it. On WSL 2, the CUDA driver used is part of the Windows driver installed on the system, and, therefore, care must be taken `not` to install this Linux driver as previously mentioned.
> 

Reviewer: ```suggestion
Normally, the CUDA toolkit for Linux comes packaged with the device driver for the GPU. On WSL 2, the CUDA driver used is part of the Windows driver installed on the system, and, therefore, care must be taken **not** to install this Linux driver.
```

Removed excess verbiage.

Shouldn't use inline code formatting for emphasis (use bold instead, if necessary).

![Terminal output showing the results of running the device query sample application.](assets/gpu-cuda/device-query.png)

> ## WSL is also supported by some AMD GPUs.
> For more information on AMD, [this might be an appropriate link to look into.](https://rocm.docs.amd.com/projects/radeon/en/latest/docs/install/wsl/howto_wsl.html)
> 
> 

Reviewer: ```suggestion
## AMD support for WSL

While this guide focuses on NVIDIA, WSL is also supported by some AMD GPUs.

If you need to use WSL with an AMD GPU, refer to the official AMD documentation.

The documentation includes a dedicated [guide for using AMD Radeon GPUs with Ubuntu on WSL](https://rocm.docs.amd.com/projects/radeon/en/latest/docs/install/wsl/howto_wsl.html).
```

I don't think a single sentence warrants a standalone section, so I fleshed this out a bit. For example, their docs specifically focus on using the Ubuntu distro on WSL, so we can say that.

I also tightened up the header and removed the period.

* [Setting up WSL for Data Science](https://ubuntu.com/blog/wsl-for-data-scientist)
* [Ubuntu WSL for Data Scientists Whitepaper](https://ubuntu.com/engage/ubuntu-wsl-for-data-scientists)
* [NVIDIA's CUDA Post Installation Actions](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/)

> * [Install Ubuntu on WSL2](../howto/install-ubuntu-wsl2.md)

Reviewer: ```suggestion
* [Install Ubuntu on WSL 2](howto::install-ubuntu-wsl)
```
Add space in "WSL 2" and use cross-ref target.
